scripts/provisioning/roles/lustre-client/tasks/main.yml

```diff
@@ -55,10 +55,10 @@
     - { version: 2.10.8, enabled: '{{ (redhat_release is version("7.6", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.4, enabled: '{{ (redhat_release is version("7.7", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.5, enabled: '{{ (redhat_release is version("7.8", "=")) | ternary("yes", "no") }}' }
-    - { version: 2.12.6, enabled: '{{ (redhat_release is version("7.9", "=")) | ternary("yes", "no") }}' }
-    - { version: 2.12.6, enabled: '{{ (redhat_release is version("8.3", "=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.6, enabled: '{{ (redhat_release is version("7.9", "="))
+                                   or (redhat_release is version("8.3", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.7, enabled: '{{ (redhat_release is version("8.4", "=")) | ternary("yes", "no") }}' }
-    - { version: 2.12.8, enabled: '{{ (redhat_release is version("8.5", ">=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.8, enabled: '{{ (redhat_release is version("8", ">=")) | ternary("yes", "no") }}' }
```

Comment:
This change may enable two version for centos 8.3 and centos 8.4

Reply:
It doesn't, I checked.

Reply:
Let me try something else...

Reply:
Try now?